### PR TITLE
[FR] Add `Skinning/skin.ini`

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal for the French translation of the wiki page `skin.ini`.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

*Feel free to merge if the delay is too long for a review by a native.*

### CODE REVIEW

File: wiki/Skinning/skin.ini/fr.md [hunk 1]
```diff
@@ -0,0 +1,798 @@
...
+Vous trouverez ci-dessous le journal des modifications apportées au skin.
+Le numéro de version, vu dans les en-têtes, est celui qui sera utilisé dans la commande `Version` pour le fichier `skin.ini`.
+
+### 1.0
+
+**Skin original 2007 - 2013**
+
+Si votre `skin.ini` ne spécifie pas de `Version`, il prendra par défaut cette version.
+
+- Les numéros des hit circle font partie des explosions de hitburst..
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:21]:
```suggestion
- Les numéros des hit circle font partie des explosions de hitburst.
```

File: wiki/Skinning/skin.ini/fr.md [hunk 2]
```diff
@@ -0,0 +1,798 @@
...
+
+- `taiko-bar-right.png` et `taiko-bar-right-glow.png` se trouvent directement sous `taiko-bar-left.png`.
+- Changements de position du tambour Taiko (permet des zones plus larges).
+
+### 2.2
+
+**Modifications de l'interface/UI**
+
+- Prise en charge des vignettes.
+  - Doit être activé par l'utilisateur dans les [options](/wiki/Options) et être supporté par le skin dans le fichier [skin.ini](/wiki/Skinning/skin.ini).
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:57]:
```suggestion
  - Doit être activé par l'utilisateur dans les [options](/wiki/Options) et être pris en charge par le skin dans le fichier [skin.ini](/wiki/Skinning/skin.ini).
```

File: wiki/Skinning/skin.ini/fr.md [hunk 3]
```diff
@@ -0,0 +1,798 @@
...
+- Modifie l'affichage du star ranting en mettant à l'échelle `star.png` au lieu de `star.png` partiellement élargi.
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:58]:
```suggestion
- Modifie l'affichage du star rating en mettant à l'échelle `star.png` au lieu de `star.png` partiellement élargi.
```

File: wiki/Skinning/skin.ini/fr.md [hunk 4]
```diff
@@ -0,0 +1,798 @@
...
+- Optimisation de l'alignement du texte du panneau.
+
+### 2.3
+
+**Changements d'osu!catch**
+
+- `fruit-ryuuta.png` ne fonctionnera plus à partir de maintenant.
+- Nouveaux états de l'attrapeur (et images).
+- Nouveaux combobursts spécifiques à osu!catch (`comboburst-fruits.png`).
+  - Les combobursts  d'osu! ne sera plus utilisé pour osu!catch.
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:68]:
```suggestion
  - Les combobursts d'osu! ne sera plus utilisé pour osu!catch.
```

File: wiki/Skinning/skin.ini/fr.md [hunk 5]
```diff
@@ -0,0 +1,798 @@
...
+
+### 2.4
+
+**Ajustement de l'échelle du stage d'osu!mania**
+
+- Réduit le nombre de compteurs de combo et de coups.
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:74]:
```suggestion
- Rétrécit le compteur de combo et les hitbursts.
```

File: wiki/Skinning/skin.ini/fr.md [hunk 6]
```diff
@@ -0,0 +1,798 @@
...
+- Ajout de `comboburst-mania.png`.
+  - Affichages sur le côté droit de la scène.
+
+### latest
+
+**Toujours la version la plus récente**
+
+Si votre dossier de skin ne contient pas de fichier `skin.ini`, il prendra par défaut cette version.
+
+- **Ne l'utilisez jamais** pour distribuer des skins ! (si une nouvelle version de skin sort, un skin dont la version est réglée sur `latest` pourrait se casser).
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:107]:
```suggestion
- **Ne l'utilisez jamais** pour distribuer des skins ! (si une nouvelle version de skin sort, un skin dont la version est réglée sur `latest` pourrait ne pas fonctionner correctement).
```

File: wiki/Skinning/skin.ini/fr.md [hunk 7]
```diff
@@ -0,0 +1,798 @@
...
+- Utilise toujours la dernière version supportée par le jeu.
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:108]:
```suggestion
- Utilisera toujours la dernière version prise en charge par le jeu.
```

File: wiki/Skinning/skin.ini/fr.md [hunk 8]
```diff
@@ -0,0 +1,798 @@
...
+
+### Utilisateur
+
+**Pas une version, mais toujours forcer [latest](#latest)**
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:112]:
```suggestion
**Pas une version, mais forcera toujours [latest](#latest)**
```

File: wiki/Skinning/skin.ini/fr.md [hunk 9]
```diff
@@ -0,0 +1,798 @@
...
+- Le dossier skin **doit** être nommé `User`.
+- Utilisez cette méthode si vous ne souhaitez modifier que quelques éléments (par exemple, le curseur ou les chiffres, etc.).
+- Ne nécessite pas le fichier `skin.ini`.
+- Ce dossier forcera toujours `Version : latest`.
+
+## Notes
+
+Avant de consulter les commandes `skin.ini` ci-dessous, voici quelques remarques.
+
+### 1's and 0's
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:124]:
```suggestion
### 1 et 0
```

File: wiki/Skinning/skin.ini/fr.md [hunk 10]
```diff
@@ -0,0 +1,798 @@
...
+
+Certaines commandes n'acceptent qu'une valeur booléenne (une valeur `true` ou `false`). Lors du skinning, osu! est configuré pour ne reconnaître qu'un `1` (un) comme `vrai` et un `0` (zéro) comme `faux`.
+
+Voici un exemple classique :
+
+| `SliderBallFlip: 0` | `SliderBallFlip: 1` |
+| :-: | :-: |
+| ![](img/Sliderball_flip-0.gif) | ![](img/Sliderball_flip-1.gif) |
+
+Notez que Reisen, le sliderball, ne se retourne pas lorsque `0` est utilisé. Cependant, Reisen se retourne quand un `1` est utilisé. En fonction du sprite utilisé, vous obtiendrez soit une balle glissante qui fait un moonwalk, soit une balle qui tourne sur elle-même.
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:136]:
```suggestion
Notez que Reisen, la sliderball, ne se retourne pas lorsque `0` est utilisé. Cependant, Reisen se retourne quand un `1` est utilisé. En fonction du sprite utilisé, vous obtiendrez soit une sliderball qui fait du moonwalk, soit une sliderball qui se retourne.
```

File: wiki/Skinning/skin.ini/fr.md [hunk 11]
```diff
@@ -0,0 +1,798 @@
...
+
+### Nombres et nombres entiers
+
+Les tableaux ci-dessous peuvent énumérer soit un *nombre*, un *entier* ou un *entier positif*.
+
+When viewing these tables:
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:142]:
```suggestion
Concernant ces tableaux :
```

File: wiki/Skinning/skin.ini/fr.md [hunk 12]
```diff
@@ -0,0 +1,798 @@
...
+
+Voici un exemple classique :
+
+| `SliderBallFlip: 0` | `SliderBallFlip: 1` |
+| :-: | :-: |
+| ![](img/Sliderball_flip-0.gif) | ![](img/Sliderball_flip-1.gif) |
+
+Notez que Reisen, le sliderball, ne se retourne pas lorsque `0` est utilisé. Cependant, Reisen se retourne quand un `1` est utilisé. En fonction du sprite utilisé, vous obtiendrez soit une balle glissante qui fait un moonwalk, soit une balle qui tourne sur elle-même.
+
+### Nombres et nombres entiers
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:138]:
```suggestion
### Nombres et entiers
```

File: wiki/Skinning/skin.ini/fr.md [hunk 13]
```diff
@@ -0,0 +1,798 @@
...
+
+Les tableaux ci-dessous peuvent énumérer soit un *nombre*, un *entier* ou un *entier positif*.
+
+When viewing these tables:
+
+- *nombres* signifie un nombre **entier** ou **décimal** (par exemple `1.5`, `4.295`, `2`, `3.0`).
+- *entier* signifient **uniquement** des nombres entiers (par exemple : `-13`, `-632`, `135` , `9`).
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:145]:
```suggestion
- *entier* signifie **uniquement** des nombres entiers (par exemple : `-13`, `-632`, `135` , `9`).
```

File: wiki/Skinning/skin.ini/fr.md [hunk 14]
```diff
@@ -0,0 +1,798 @@
...
+  - *positive integer* signifie **nombre entier positif** uniquement (par exemple `376`, `22`, ou `5`).
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:146]:
```suggestion
  - *entier positif* signifie **nombre entier positif** uniquement (par exemple `376`, `22`, ou `5`).
```

File: wiki/Skinning/skin.ini/fr.md [hunk 15]
```diff
@@ -0,0 +1,798 @@
...
+
+Notez que Reisen, le sliderball, ne se retourne pas lorsque `0` est utilisé. Cependant, Reisen se retourne quand un `1` est utilisé. En fonction du sprite utilisé, vous obtiendrez soit une balle glissante qui fait un moonwalk, soit une balle qui tourne sur elle-même.
+
+### Nombres et nombres entiers
+
+Les tableaux ci-dessous peuvent énumérer soit un *nombre*, un *entier* ou un *entier positif*.
+
+When viewing these tables:
+
+- *nombres* signifie un nombre **entier** ou **décimal** (par exemple `1.5`, `4.295`, `2`, `3.0`).
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:144]:
```suggestion
- *nombre* signifie un nombre **entier** ou **décimal** (par exemple `1.5`, `4.295`, `2`, `3.0`).
```

File: wiki/Skinning/skin.ini/fr.md [hunk 16]
```diff
@@ -0,0 +1,798 @@
...
+  - Question : Quelle couleur faut-il ajouter à star2 pendant les pauses ?
+  - Valeur : *RGB*
+  - Par défaut : `255,182,193`
+  - Notes :
+    - couleur de `star2` pendant les pauses
+
+## \[Fonts\]
+
+- `HitCirclePrefix:`
+  - Question : Quel est le préfixe utilisé pour les numéros des hitcircle ?
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:377]:
```suggestion
  - Question : Quel est le préfixe utilisé pour les numéros des hit circles ?
```

File: wiki/Skinning/skin.ini/fr.md [hunk 17]
```diff
@@ -0,0 +1,798 @@
...
+  - Valeur : texte *(préfixe du chemin/du nom de fichier)*
+  - Par défaut : `default`
+  - Notes :
+    - Vous pouvez utiliser un chemin personnalisé
+- `HitCircleOverlap:`
+  - Question : De combien de pixels les numéros d'hitcircle doivent-ils se chevaucher ?
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:383]:
```suggestion
  - Question : De combien de pixels les numéros d'hit circle doivent-ils se chevaucher ?
```

File: wiki/Skinning/skin.ini/fr.md [hunk 18]
```diff
@@ -0,0 +1,798 @@
...
+    - Il s'agit de la couleur des images qui suivent l'attrapeur après avoir obtenu le fruit de l'hyperdash.
+
+## \[Mania\]
+
+**Notes :**
+
+- Si vous voulez personnaliser entièrement la scène osu!mania, utilisez le skin version `2.5` ou supérieure.
+- Les commandes qui demandent où positionner quelque chose horizontalement sont basées sur une hauteur de 480 pixels.
+- Lorsqu'une commande est dépendante du nombre total de touches :
+  - en utilisant trop et les extras sont ignorés.
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:440]:
```suggestion
  - les touches en trop et les extras sont ignorés.
```

File: wiki/Skinning/skin.ini/fr.md [hunk 19]
```diff
@@ -0,0 +1,798 @@
...
+  - pas assez et les valeurs manquantes utiliseront des valeurs par défaut.
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:441]:
```suggestion
  - ne pas utiliser assez de touches et les valeurs manquantes utiliseront des valeurs par défaut.
```

File: wiki/Skinning/skin.ini/fr.md [hunk 20]
```diff
@@ -0,0 +1,798 @@
...
+- **Chaque keycount  *doit* commencer une nouvelle section**, cela devrait ressembler à ceci (mais avec des commandes réelles) :
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:442]:
```suggestion
- **Chaque keycount *doit* commencer une nouvelle section**, cela devrait ressembler à ceci (mais avec des commandes réelles) :
```

File: wiki/Skinning/skin.ini/fr.md [hunk 21]
```diff
@@ -0,0 +1,798 @@
...
+  - Valeur : *nombres*
+  - Par défaut : `40`
+  - Notes : La distance des deux étapes, si elle est fractionnée.
+- `SeparateScore:`
+  - Question : Le hitburst doit-il être affiché uniquement sur la scène où il a été marqué ?
+  - Valeur : `0` ou `1`
+  - Par défaut : `1`
+  - Notes :
+    - `0` = s'affiche sur les deux scènes en même temps
+    - `1` = s'affiche lorsqu'il est frappé sur une scène spécifique
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:591]:
```suggestion
    - `1` = s'affiche lorsqu'il est touché sur une scène spécifique
```

File: wiki/Skinning/skin.ini/fr.md [hunk 22]
```diff
@@ -0,0 +1,798 @@
...
+    - `2` = `Both` (aléatoire)
+    - Vous pouvez utiliser les mots ou les valeurs, les deux sont acceptés.
+    - Les combobursts seront retournés sur la moitié droite de la scène.
+- `SplitStages:`
+  - Question : La scène doit-elle être divisée en deux ?
+  - Valeur : `0` ou `1`
+  - Notes :
+    - Cette valeur est forcée, si elle est définie.
+    - `0` = pas de fractionnement / SP forcé
+    - Chaque keycount  supérieur à 1 peut être divisé (ou fusionné, si le nombre est supérieur à 9).
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:579]:
```suggestion
    - Chaque keycount supérieur à 1 peut être divisé (ou fusionné, si le nombre est supérieur à 9).
```

File: wiki/Skinning/skin.ini/fr.md [hunk 23]
```diff
@@ -0,0 +1,798 @@
...
+- Nouvelles commandes :
+  - `KeyFlipWhenUpsideDown` et `NoteFlipWhenUpsideDown`
+  - `NoteBodyStyle` (étirement, cascade du haut, cascade du bas)
+  - `LightingNWidth` et `LightingLWidth`
+
+### 2.6
+
+**Permettre le skinning des flèches par type de flèche**
+
+- Ajout de  `arrow-generic.png`, `arrow-warning.png` et `arrow-pause.png`.
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:91]:
```suggestion
- Ajout de `arrow-generic.png`, `arrow-warning.png` et `arrow-pause.png`.
```

File: wiki/Skinning/skin.ini/fr.md [hunk 24]
```diff
@@ -0,0 +1,798 @@
...
+- *nombres* signifie un nombre **entier** ou **décimal** (par exemple `1.5`, `4.295`, `2`, `3.0`).
+- *entier* signifient **uniquement** des nombres entiers (par exemple : `-13`, `-632`, `135` , `9`).
+  - *positive integer* signifie **nombre entier positif** uniquement (par exemple `376`, `22`, ou `5`).
+- *liste séparée par des virgules avec des entiers positifs* est littéralement une liste d'entiers positifs séparés par des virgules (par exemple `1, 2, 3, 55`).
+
+### RGB et RGB(a)
+
+Quelques commandes peuvent demander une couleur au format *RGB* ou *RGB(a)*.
+
+- Pour RGB, le format ressemble à ceci :  `R, G, B` où `R` est le rouge, `G` est le vert et `B` est le bleu.
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:153]:
```suggestion
- Pour RGB, le format ressemble à ceci : `R, G, B` où `R` est le rouge, `G` est le vert et `B` est le bleu.
```

File: wiki/Skinning/skin.ini/fr.md [hunk 25]
```diff
@@ -0,0 +1,798 @@
...
+  - Notes :
+    - `StageHint`, `LightingN`, et `LightingL` sont dessinés à cette position.
+- `LightPosition:`
+  - Question : A quelle hauteur doivent être tirées les lumières de la scène ?
+  - Valeur : *entier*
+  - Par défaut : `413`
+  - Notes :
+    - Ceci est seulement pour `StageLight`.
+- `ScorePosition:`
+  - Question : A quelle hauteur les hitbursts  doivent-ils apparaître ?
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:534]:
```suggestion
  - Question : A quelle hauteur les hitbursts doivent-ils apparaître ?
```

File: wiki/Skinning/skin.ini/fr.md [hunk 26]
```diff
@@ -0,0 +1,798 @@
...
+    - Nécessite la version `2.5` ou supérieure du skin.
+    - Spécifique à la colonne (pour les head parts)
+- `NoteFlipWhenUpsideDown#L:`
+  - Question : Le hold note body de la colonne doit-il être retourné lorsque la scène est retournée ?
+  - Valeur : `0` ou `1`
+  - Notes :
+    - Nécessite la version `2.5` ou supérieure du skin.
+    - Spécifique à la colonne (pour les length parts)
+- `NoteFlipWhenUpsideDown#T:`
+  - Question : La  hold note tail de la colonne doit-elle être inversée lorsque la scène est inversée ?
```
Comment (Ganondorf-1) [inline wiki/Skinning/skin.ini/fr.md:651]:
```suggestion
  - Question : La hold note tail de la colonne doit-elle être inversée lorsque la scène est inversée ?
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Comment by DoryanR** [comment]
@Ganondorf-1 recheck please

**Review by TicClick — APPROVED** [review]
merging with the assumption that one review was enough

**Pull request merged**